book/04-git-server/sections/protocols.asc

```diff
@@ -53,7 +53,7 @@ If you and a co-worker are working on the same project and they want you to chec
 
 ===== The Cons
 
-The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access.
+The cons of this method are that shared access is generally more difficult to set up and reach from multiple locations than basic network access, and it is easier to damage the repository.
```

Comment:
What about adding this as a paragraph below this one? Adding it as a clause here seems to interrupt the point being made, and I think there's more than just half a sentence of "con" here.